Makefile.toml

```diff
@@ -0,0 +1,65 @@
+extend = "./scripts/cargo-helpers.toml"
+
+# component builds
+[tasks.all]
+dependencies = [ "tools", "backend", "frontend" ]
+workspace = false
+
+[tasks.frontend]
+workspace = false
+dependencies = [ "atomify" ]
+command = "cargo"
+args = [ "make", "--cwd", "client/", "--makefile", "Makefile.toml", "all" ]
+
+[tasks.tools]
+workspace = false
+dependencies = [ "atomify", "bone_cake" ]
+
+[tasks.backend]
+workspace = false
+dependencies = [ "game-engine", "game-server" ]
+
+# package builds
+[tasks.game-server]
+workspace = false
+command = "cargo"
+args = [ "make", "exec-${BUILD_ENV}", "--", "build", "-p", "game-server" ]
+
+[tasks.game-engine]
+workspace = false
+command = "cargo"
+args = [ "make", "exec-${BUILD_ENV}", "--", "build", "-p", "rask_game_engine" ]
+
+[tasks.atomify]
+workspace = false
+command = "cargo"
+args = [ "make", "exec-${BUILD_ENV}", "--", "build", "-p", "atomify" ]
+
+[tasks.bone_cake]
+workspace = false
+command = "cargo"
+args = [ "make", "exec-${BUILD_ENV}", "--", "build", "-p", "bone_cake" ]
+
+# dev tools
+[tasks.serve-frontend]
+command = "./scripts/dev-server"
+workspace = false
+args = [ "${@}" ]
+dependencies = [ "frontend" ]
+
+[tasks.serve-backend]
+workspace = false
+command = "cargo"
+args = [ "make", "exec-${BUILD_ENV}", "--", "run", "-p", "game-server" ]
+
+# upstream `ci-flow` task appears to have problems with wasm
+[tasks.ci-flow]
+workspace = false
+dependencies = [ "clean", "all", "check-format", "test" ]
```

Comment:
Damn, I forgot to push the new `test` target :man_facepalming: .
Will fix that later, the `cargo make test` target provided by `cargo-make` itself breaks with some WASM outputs, I'm not sure yet what's the best/simplest fix for that :)